docs/database/mysql.md

```diff
@@ -0,0 +1,24 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: MSSQL
+--->
+
+# Semantic Conventions for MySQL
+
+**Status**: [Experimental][DocumentStatus]
+
+The Semantic Conventions for the [MySQL](https://www.mysql.com/) extend and override the [Database Semantic Conventions](database-spans.md)
+that describe common database operations attributes in addition to the Semantic Conventions
+described on this page.
+
+`db.system` MUST be set to `"mysql"`.
+
+## Connection-level attributes
+
+<!-- semconv db.mysql(tag=connection-level-tech-specific) -->
+| Attribute  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+| [`db.jdbc.driver_classname`](database-spans.md) | string | The fully-qualified class name of the [Java Database Connectivity (JDBC)](https://docs.oracle.com/javase/8/docs/technotes/guides/jdbc/) driver used to connect. | `org.postgresql.Driver`; `com.microsoft.sqlserver.jdbc.SQLServerDriver` | Recommended |
+| `db.mysql.instance.address` | string | The hostname of the MySQL server instance bound to the current connection. This is useful in cases where the client is connecting to a MySQL server via a proxy or in a clustered environment. This client may obtain this value using queries like `select @@hostname`. | `mysql-e26b99z.example.com` | Recommended: If different from the `server.address` |
```

Comment:
Do you, given the proposals in #321 (e. g. https://github.com/open-telemetry/semantic-conventions/pull/321#issuecomment-1720609696), that `server.address` could be sufficient to model this?

Reply:
Got it, so `db.mysql.instance.address` will identify the node inside the MySQL cluster. And `server.address` will be the address the user connects to.

Reply:
Yes!